nbs/sample_question.ipynb

```diff
@@ -0,0 +1,155 @@
+{
```

Comment:
<p>The problem here is that all of your data is already relational, you didnt need any ie functions here, this example could be done in normal datalog.</p><p> </p><p>Make the input data more messy in a real life kind of way (both real texts, extra structure we need to ignore from jsons etc...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/spanner_workbench/pull/130/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>